### DISCUSSION TITLE
Problem installing a new IDH node with the SO installer 2.3.140

### DISCUSSION THREAD

**Discussion by b-u-g-s** [Security-Onion-Solutions/securityonion#8508] [General] [answered]
Hi, 
I am trying to create an IDH node with the SO 2.3.140 installer, but whatever I select when I boot the ISO, I only sendup with a standard installation and the setup menu does not let me select an IDH installation.

More info:
- I have a manager installed in a VM as a standalone on an HYPER-V server
- I then create a new VM on the same HYPER-V server and give it the minimum required (1Gb, 2 CPU, 100Gb, 1x LAN)
- I boot the new VM with the SO 2.3.140 ISO mounted
- I can see the boot menu, I select INSTALL SECURITY ONION 2.3.140 IDH Node

This goes through what looks like a "standard" installation (meaning quite long with 1007 or so modules to install (which doesn't seem right for just a honeypot, but maybe that's normal)
The VM reboots
I log in
I get the setup menu, but that's the standard setup menu which only allow me to install an EVAL, STANDALONE, etc...
If I select OTHER, I don't get any IDH option

Looks to me as if the ISO boot menu does not actually take my choice in consideration and just install the standard SO 2.3.140

I also tried the install IDH node in basic graphics mode and I was getting the same install process so I cancelled.

The doc:
https://docs.securityonion.net/en/2.3/idh.html#installation
Seems to indicate you can just use the SO ISO 2.3.140 to install an IDH (and the ISO boot menu looks like it can).

Am I missing something here?
Thanks.
Bugs

**Comment by isaacgolding** [comment, marked as answer]
Is your VM defaulting back to the installer ISO to boot from?   If so unmount the ISO and boot.

**Reply by b-u-g-s** [reply]
Unfortunately not, it automatically removed the DVD/ISO after the initial install. So I am booting to the installed SO instance for the post installation setup phase. 
first time I try to install the SO honeypot so never tried on older SO version.Aanyone else installed one on the .140?
I can see from the install boot prompt that it seems to be using a different script.
Initially I thought I made a mistake and did not select the right install « line », but I have now done it 3 more times and I am sure I am selecting installing a honeypot node, even try with the basic graphic option to see if it would make a difference. 
not for me.

**Reply by isaacgolding** [reply]
For what it's worth I had problems setting up my IDH on my VM (proxmox) and I ended up going with 8 FGB of ram and 4CPUs just to help speed up the process.   And then once it was all settled and sync'd and running right I went back and throttled it back to min specs.  I'm spinning up a VM now.  Gimmme 20 min.

**Reply by isaacgolding** [reply]
Ok you are going down the wrong install path.    when you are presented with the basic install options of EVAL,STAND,DISTR,IMPORT,OTHER go with DISTRIBUTED, then choose "existing deployment",  then in the following screen you can pick IDH.

**Reply by b-u-g-s** [reply]
> For what it's worth I had problems setting up my IDH on my VM (proxmox) and I ended up going with 8 FB of ram and 4CPUs just to help speed up the process. And then once it was all settled and sync'd and running right I went back and throttled it back to min specs. I'm spinning up a VM now. Gimmme 20 min.

Thanks, that's a good tip!

**Reply by b-u-g-s** [reply]
> Ok you are going down the wrong install path. when you are presented with the basic install options of EVAL,STAND,DISTR,IMPORT,OTHER go with DISTRIBUTED, then choose "existing deployment", then in the following screen you can pick IDH.

Thank you so much! I would have never thought of that! :)
now installing my first IDH...

Thanks for checking this! I

**Reply by dougburks** [reply]
For others that might find this discussion, the IDH setup process is documented here:
https://docs.securityonion.net/en/2.3/idh.html#screenshots

**Reply by b-u-g-s** [reply]
Thanks Doug. For me there was one screenshot missing on that page which would have helped: A first screenshot of when you select DISTRIBUTED. It might be obvious for some, but if this is the first time you setup a node it might not be obvious. 
Also, it would be great if when selecting "install IDH" at boot, you are then presented with a different menu than the standard install. From the doc I understand it partitions the disk differently, but is there a case for installing anything else than a distributed IDH node after selecting "install IDH" at boot? would someone select standalone or eval?

**Reply by dougburks** [reply]
> For me there was one screenshot missing on that page which would have helped: A first screenshot of when you select DISTRIBUTED. It might be obvious for some, but if this is the first time you setup a node it might not be obvious.

From https://docs.securityonion.net/en/2.3/idh.html#screenshots:
_Run Setup, choose to add a new node to your distributed deployment, and then select the IDH option._

If you would like to suggest alternate wording to make that more obvious, I'm happy to update it.

**Reply by b-u-g-s** [reply]
What about something like that:
"Run Setup, select the DISTRIBUTED install submenu, choose to add a new node to your existing deployment, and then select the IDH option."

**Reply by dougburks** [reply]
Done.
